features/include/pcl/features/rsd.h

```diff
@@ -64,7 +64,7 @@ namespace pcl
     const int cols = histograms2D.at(0).cols();
 
     typename PointCloud<Histogram<N> >::VectorType::iterator it = histogramsPC.points.begin ();
-    BOOST_FOREACH (Eigen::MatrixXf h, histograms2D)
+    for (Eigen::MatrixXf h : histograms2D)
```

Comment:
Looks like a case for `std::transform`

Reply:
I'm not sure what this code is really doing, so I don't want to do this in this PR.